app/views/entities/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<% provide(:title, "Transactions") %>
+<div class='card'>
+  <div class="row border border-light shadow-sm">
+    <div class="col-9">
+        <p class="text-muted fs-5 py-3 ps-4">Category <%= @group.name %>:</p>
+    </div>
+    <div class="col-3">
+        <p class="card-text py-3"><small class="fs-5 p-3">$<%= GroupEntity.where("group_id = #{@group.id}").joins(:entity).sum("amount")%></small></p> 
+    </div>
+  </div>
+  <% if @group.group_entities.empty? %>
+      <p class="text-muted fs-5 py-3 ps-4">No transactions found in this category.</p>
+  <% end %>
+  <% if @group.user_id == @user %>
+    <% @group.group_entities.reverse_order.each do |entity|%>
+        <div class="card-body container">
+          <div class="card justify-content-center shadow rounded-3">
+            <div class="row g-0">
+              <div class="col-3">
+                <img src="<%= @group.icon %>" class="img-fluid rounded-start p-4" alt="...">
+              </div>
+              <div class="col-6 d-flex align-items-center">
+                <div class="card-body ">
+                  <h5 class="card-title text-secondary"><%= Entity.find(entity.entity_id).name %></h5>
+                  <p class="card-text mt-4 text-black-50"><%= Entity.find(entity.entity_id).created_at.strftime("%d %b %Y") %></p>
+                </div>
+              </div>
+              <div class="col-3 d-flex align-items-center">
+                <div class="card-body">
+                  <p class="card-text"><small class="fs-5 p-3">$<%= Entity.find(entity.entity_id).amount %></small></p> 
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+    <% end %>
+  <% end %>
+</div>
```

Comment:
- [ ]  We don't need to add any logic inside the .erb file, we use the helper folders to create methods with logic and called inside the .erb file. You can refer to this comment to modify the files that have logic inside the .erb file.

Read [this](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make) for more information